modules/installation/proc-configuring-an-rhdh-instance-with-tls-in-kubernetes.adoc

```diff
@@ -0,0 +1,104 @@
+[id="proc-configuring-an-rhdh-instance-with-tls-in-kubernetes_{context}"]
+= Configuring an {product-very-short} instance with a TLS connection in Kubernetes
+
+You can configure an {product-very-short} instance with a Transport Layer Security (TLS) connection in a Kubernetes cluster, such as an Azure Red Hat OpenShift (ARO) cluster, any other cloud provider cluster, or your own cluster with proper configuration. However, You must use a public Certificate Authority (CA)-signed certificate to configure your Kubernetes cluster.
+
+.Prerequisites
+
+* You have set up an Azure Red Hat OpenShift (ARO) cluster with a public CA-signed certificate. For more information about obtaining CA certificates, refer to your vendor documentation.
+* You have created a namespace and setup a service account with proper read permissions on resources. For more information, see link:https://backstage.io/docs/features/kubernetes/configuration#role-based-access-control[Role-based access control].
+* You have obtained the secret and the service CA certificate associated with your service account.
+* You have created some resources and added labels to them so they can be discovered by the Kubernetes plugin. For more information, see link:https://backstage.io/docs/features/kubernetes/configuration#common-backstageiokubernetes-id-label[Common `backstage.io/kubernetes-id` label].
+
+.Procedure
+
+. Enable the Kubernetes plugins in the `dynamic-plugins-rhdh.yaml` ConfigMap file:
++
+[source,yaml]
+----
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: dynamic-plugins-rhdh
+data:
+  dynamic-plugins.yaml: |
+    includes:
+      - dynamic-plugins.default.yaml
+    plugins:
+      - package: ./dynamic-plugins/dist/backstage-plugin-kubernetes-backend-dynamic
+        disabled: false <1>
+      - package: ./dynamic-plugins/dist/backstage-plugin-kubernetes
```

Comment:
The kubernetes frontend plugin is TP.  Should we be documenting TP plugins?  Maybe using the topology plugin is an alternative?

Reply:
Is this the plugin `./dynamic-plugins/dist/janus-idp-backstage-plugin-topology-dynamic` you are talking about? Thanks

Reply:
ok, let's highlight the fact that the kubernetes frontend plugin is TP.   The topology plugin is an alternative that can be a permanent fixture since it is GA level.